### PR TITLE
Added support for maxUnavailable

### PR DESCRIPTION
**Pull request by geetikakay**
Signed-off-by: Geetika Kapoor <gkapoor@redhat.com>

##### Short description:
Support for maxunavailable for node network configuration policy

### CODE REVIEW

File: ocp_resources/node_network_configuration_policy.py [hunk 1]
```diff
@@ -147,6 +149,9 @@ def to_dict(self):
 
             res["spec"]["desiredState"]["interfaces"] = self.desired_state["interfaces"]
 
+        if self.max_unavailable:
+            res["spec"]["maxUnavailable"] = self.max_unavailable
```
Comment (rnetser) [inline ocp_resources/node_network_configuration_policy.py:153]: res.setdefault("spec", {})....

**Review by rnetser — CHANGES_REQUESTED** [review]

File: ocp_resources/node_network_configuration_policy.py (hunk 1 above)
Comment (geetikakay) [reply ocp_resources/node_network_configuration_policy.py]: ack @rnetser

**Comment by myakove** [comment]
Please update the commit msg, support for which resource.

**Comment by myakove** [comment]
and rebase please.

**Comment by geetikakay** [comment]
@myakove @rnetser please review. I have  fixed tox failures.

**Comment by geetikakay** [comment]
/verified

Verified in downstream patch as dependency

**Comment by geetikakay** [comment]
Verified in downstream patch as dependency

**Comment by myakove** [comment]
> Verified in downstream patch as dependency

thanks, No need to mention d/s here :)

**Pull request merged**